source/grid/grid_tools.cc

```diff
@@ -1122,6 +1123,67 @@ namespace GridTools
   }
 
 
+
+  template <int dim, template <int, int> class MeshType, int spacedim>
+  unsigned int
+  find_closest_vertex (const Mapping<dim,spacedim>  &mapping,
+                       const MeshType<dim,spacedim> &mesh,
+                       const Point<spacedim>        &p,
+                       const std::vector<bool>      &marked_vertices)
+  {
+    // Take a shortcut in the simple case.
+    if (mapping.preserves_vertex_locations() == true)
+      return find_closest_vertex(mesh, p, marked_vertices);
```

Comment:
given this can you add a note to the existing `find_closest_vertex` function stating that it only takes the locations of vertices as stated in the triangulation, but that there is another function that allows displacing vertices via a mapping?

Reply:
done.